addon/components/frost-chart-axis-tick-svg.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Component definition for the frost-chart-axis-tick-svg component
+ */
+
+import Ember from 'ember'
+const {String: EmberString, isNone, run} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {Component} from 'ember-frost-core'
+import {PropTypes} from 'ember-prop-types'
+
+export default Component.extend({
+
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+
+  attributeBindings: ['transform'],
+  tagName: 'g',
+
+  // == PropTypes =============================================================
+
+  propTypes: {
+    // options
+    axis: PropTypes.string.isRequired,
+    coordinate: PropTypes.number
+
+    // state
+  },
+
+  getDefaultProps () {
+    return {
+      // options
+      coordinate: null
+
+      // state
+    }
+  },
+
+  // == Computed Properties ===================================================
+
+  @readOnly
+  @computed('coordinate')
+  transform (coordinate) {
+    if (isNone(coordinate)) {
+      return EmberString.htmlSafe('')
+    }
+
+    // calc is added to align the middle of the tick with the location
+    let x = 0
```

Comment:
Really really minor, but I prefer to use `const` whenever possible, so we could restructure this as

```
const axis = this.get('axis')
const x = axis === 'x' ? coordinate : 0
const y = axis === 'y' ? coordinate : 0
```